.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "editor.formatOnSave": true,
+  "eslint.validate": ["javascript"]
+}
```

Comment:
This will make sure format on save is enabled, even if global settings are different

Reply:
I'd actually prefer not to force developers to have this turned on. We of course want to prevent devs from submitting pull requests that are not properly formatted, but there are better ways imo. For instance using git hooks (that can be bypassed) or running eslint in the Github build pipeline.